src/main/java/hudson/plugins/build_timeout/BuildTimeoutWrapper.java

```diff
@@ -41,7 +42,8 @@
  * @author Kohsuke Kawaguchi
  */
 public class BuildTimeoutWrapper extends BuildWrapper {
-    
+
+    @SuppressFBWarnings(value = "MS_SHOULD_BE_FINAL", justification = "Diagnostic fields are left mutable so that groovy console can be used to dynamically turn/off probes.")
     public static long MINIMUM_TIMEOUT_MILLISECONDS = Long.getLong(BuildTimeoutWrapper.class.getName()+ ".MINIMUM_TIMEOUT_MILLISECONDS", 3 * 60 * 1000);
```

Comment:
This is a fix for the following findbugs error:
```
<BugInstance
  type='MS_SHOULD_BE_FINAL'
  priority='High'
  category='MALICIOUS_CODE'
  message='hudson.plugins.build_timeout.BuildTimeoutWrapper.MINIMUM_TIMEOUT_MILLISECONDS isn&apos;t final but should be'
```

This field is rather for testing purpose, to shorten testing duration.
We may consider to change this final and provide another way for testing.